daisy/workflow/step_create_disks.go

```diff
@@ -38,6 +38,10 @@ type CreateDisk struct {
 	// Optional description of the resource, if not specified Daisy will
 	// create one with the name of the project.
 	Description string `json:",omitempty"`
+	// Zone to create the instance in, overrides workflow zone.
```

Comment:
Not necessary, but I've been capitalizing Zone, Project, etc. so it matches the Workflow struct's fields. Take it or leave it.